content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -227,8 +227,17 @@ kubectl describe pvc <pvc_name>
 
 If the `PersistentVolumeClaim` has the status `FileSystemResizePending`, it is safe to recreate the pod using the PersistentVolumeClaim.
 
+FlexVolumes allow resize if the driver is set with the `RequiresFSResize` capability to true. 
+The FlexVolume can be resized on pod restart as 'ExpandPersistentVolume' feature gate is enabled by default. 
+
 #### Resizing an in-use PersistentVolumeClaim
 
+Expanding in-use PVCs for FlexVolumes is added in release 1.13. To enable this feature use  `ExpandInUsePersistentVolumes` and `ExpandPersistentVolumes` feature gates. The `ExpandPersistentVolumes` feature gate is already enabled by default. If the `ExpandInUsePersistentVolumes` is set, FlexVolume can be resized online without pod restart. 
+ 
+The FlexVolume plugin uses RequiresFSResize to identify and resize the Persistent Volume. The underlying driver must support resize, if not an error will be returned and Persistent Volume object will not change and the expand request will fail. FlexVolume driver must implement the method `ExpandFS` defined in the interface `FSResizableVolumePlugin` along with `ExpandableVolumePlugin` interface and its method `ExpandVolumeDevice`.
+The Flex driver should implement ExpandableVolumePlugin interface only if it is to be installed on the controller/master.
+The Flex driver should implement any volume expansion methods along with FS resize in ExpandFS method if the Flex driver is installed on the worker nodes but not on the controller or master node.
+
 {{< feature-state for_k8s_version="v1.11" state="alpha" >}}
```

Comment:
This document is user facing piece. We do not document interfaces to implement here. Such details can be moved to -  https://github.com/kubernetes/community/blob/master/contributors/devel/flexvolume.md

Reply:
A separate PR will be opened to move the text into the community page.